### ISSUE TITLE
HBOMax is now Max

### ISSUE THREAD

**Issue by ShamanSquirrel** [FrostCo/AdvancedProfanityFilter#505]
_Instructions: This template is written in Markdown. Click the "Preview" tab to see what it will look like when submitted. Fill in the applicable sections (you can erase the examples if you'd like). For more information, please see [this page](https://github.com/FrostCo/AdvancedProfanityFilter/wiki/New-Audio-Muting-Site)._

#### :movie_camera: The Site
play.max.com

#### :speech_balloon: Subtitle Element/Node
_Example:_
```html
<div class="subtitle-container">
  <p class="subtitle-text">Subtitle text</p>
</div>
```

#### :pencil: Additional notes
_Add any other context or screenshots about the request here._

**Comment by richardfrost** [comment]
Thanks for reporting this @ShamanSquirrel! I see that they have changed their captions/subtitles quite a bit but should be able to get a new version supporting it this weekend.

**Comment by sheldonhull** [comment]
I started testing with this config and the selector seems to be right but I haven't figured out why it's not hiding results yet. 

Maybe the selector bit would be useful. 


```json
{
  "play.max.com": [
    {
      "apfCaptions": true,
      "displaySelector": "div[data-testid='CueBoxContainer']",
      "displaySelectorParents": 1,
      "ignoreMutations": true,
      "mode": "watcher",
      "subtitleSelector": "div[class^=\"CaptionWindow\"]"
    }
  ]
}
```

**Comment by richardfrost** [comment]
Awesome work @sheldonhull, congrats on getting it that far! I believe I have a new config that will be a little more performant and should deal with those display artifacts. I had to make a couple code changes as well, so I'll bundle the new config with the next version. I am on track to get it released later today. But, in the meantime, feel free to use the config you shared. When the new version comes out, make sure to remove your custom code though because it will override the config included updated config for Max.

**Comment by richardfrost** [comment]
This has been included in version [3.2.2](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/3.2.2). Feel free to comment here if you have any feedback for this update.

**Comment by sheldonhull** [comment]
really appreciate it!
